shell/services/node/node_service.cc

```diff
@@ -43,7 +43,8 @@ void NodeService::Initialize(node::mojom::NodeServiceParamsPtr params) {
 
   ParentPort::GetInstance()->Initialize(std::move(params->port));
 
-  js_env_ = std::make_unique<JavascriptEnvironment>(node_bindings_->uv_loop());
+  js_env_ = std::make_unique<JavascriptEnvironment>(
+      node_bindings_->uv_loop(), NodeEnvironmentType::kNormal);
```

Comment:
Should this be `KNode` given we call `node::Stop` in the custom process exit handler below.

Reply:
Maybe I am missing some context on the change, based on the comments I understood we are disabling `only_terminate_in_safe_scope` in the `ELECTRON_RUN_AS_NODE` mode because the Node.js termination calls use some nested microtasks that triggers the CHECK in v8. What calls are responsible for this crash, are we not making the same calls during destruction of Node.js in browser and utility process ?

Reply:
We're primarily disabling `only_terminate_in_safe_scope` to match Node.js. It's not the default on `v8::Isolate`s themselves - they default this to false as well.

Reply:
Both browser process and utility process should be allowed to use Node.js profilers, v8 in both these processes are present as a result of Node.js integration, so they should align the behavior with `kNode` mode.

Reply:
ok - i'll make that change today!